src/test/java/io/stargate/sgv2/jsonapi/testresource/StargateTestResource.java

```diff
@@ -270,16 +270,12 @@ public static String getPersistenceModule() {
   }
 
   public static boolean isDse() {
```

Comment:
wonder if we can just get rid of isDse method and all usage?

Reply:
Apparently not quite: there are multiple configurations driven by this -- name of yaml config file, whether license is accepted (`.withEnv("DS_LICENSE", "accept");`).